### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by CermakM**
Added Apache 2.0 LICENSE

### CODE REVIEW

**Comment by CermakM** [comment]
PTAL and see #146

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [2020] [Fabric8 Analytics]
```
Comment (msrb) [inline LICENSE]: I am wondering if this change to the license text could confuse some of the license scanners out there. @CermakM @tisnik WDYT guys, should we stick to an unaltered license text here?

Comment (CermakM) [reply LICENSE]: This is an unaltered license. See: https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/apache-2.0.txt#L225

Comment (msrb) [reply LICENSE]: Hmm...

My understanding of that clause is that you take that [snippet](https://github.com/github/choosealicense.com/blob/gh-pages/_licenses/apache-2.0.txt#L225-L237) and use it as a "license header" in your files. You only inject your own information in those license headers, not in the full license text that lives in a dedicated file in your repository.

Example:

https://github.com/google/guava/blob/master/COPYING#L190
https://github.com/google/guava/blob/master/guava/src/com/google/common/xml/XmlEscapers.java#L1-L13

Comment (CermakM) [reply LICENSE]: Uhm, that might be a valid point.

Comment (CermakM) [reply LICENSE]: It is of course meant to be used exactly like that, but the truth is that the search engine might do an exact match to that man license file. I'll change that back to the original JIC. Thanks @msrb

**Comment by msrb** [comment]
LGTM, thanks!

**Pull request merged**